pkg/providers/aws/provider_postgres.go

```diff
@@ -779,6 +779,8 @@ func buildPostgresGenericMetricLabels(cr *v1alpha1.Postgres, clusterID, instance
 	labels["instanceID"] = instanceName
 	labels["productName"] = cr.Labels["productName"]
 	labels["strategy"] = postgresProviderName
+	labels["statusAWS"] = ""
+	labels["statusPhase"] = ""
```

Comment:
These can be removed as the generic labels should never change

Reply:
run in to cardinality issue when building info metrics https://github.com/integr8ly/cloud-resource-operator/pull/137/files#diff-8fd2a41e86b5db36e6443ed519133f75L765

Reply:
Oh right, what do you mean by cardinality issue? 

From what I've seen those `genericLabels` functions are to create static labels that do not ever change after the fact, and then only when calling the `infoLabels` functions do we add additional information such as the status

Reply:
cardinality issue applying 7 labels to 6,

Reply:
moving phase status to its own metric and restoring this to original